### PR TITLE
added put to entity fingerprints

### PR DESCRIPTION
**Pull request by guergana**
@itamargiv the only thing I am not sure is the requestbodies, should they be Item and Properties as I set them here? Or should there be Fingerprints set anywhere?

### CODE REVIEW

File: paths/entities/items/descriptions/singleLang.json [hunk 1]
```diff
@@ -11,5 +11,26 @@
...
         }
+    },
+    "put": {
+        "tags": ["items"],
+        "summary": "Add / Replace a description to an entity by language",
+        "parameters": [
+            { "$ref": "../../../../parameters.json#/entityId" },
+            { "$ref": "../../../../parameters.json#/lang" }
+        ],
+        "requestBody": { "$ref": "../../../../requests.json#/Item" },
```
Comment (itamargiv) [inline paths/entities/items/descriptions/singleLang.json:22]: In this case, The request will be a Fingerprint (where the language key is not required):
```json
{
  "language": "en",
  "value": "Some description" 
}
```

File: paths/entities/items/descriptions/singleLang.json [hunk 2]
```diff
@@ -11,5 +11,26 @@
...
+        "responses": {
+            "200": { 
+                "description": "Wikibase Entity ID",
+                "content": {
+                    "application/json": {
+                        "schema": { "type": "string" }
+                    }
+                }
+             },
```
Comment (itamargiv) [inline paths/entities/items/descriptions/singleLang.json:31]: The response should probably be the fingerprint itself, in lack of a proper ID: `EntityDescription`

File: paths/entities/items/labels/singleLang.json [hunk 3]
```diff
@@ -11,5 +11,26 @@
...
         }
+    },
+    "put": {
+        "tags": ["items"],
+        "summary": "Add / Replace a label to an entity by language",
+        "parameters": [
+            { "$ref": "../../../../parameters.json#/entityId" },
+            { "$ref": "../../../../parameters.json#/lang" }
+        ],
+        "requestBody": { "$ref": "../../../../requests.json#/Item" },
```
Comment (itamargiv) [inline paths/entities/items/labels/singleLang.json:22]: In this case, The request will be a Fingerprint (where the language key is not required):
```json
{
  "language": "en",
  "value": "Some label" 
}
```

File: paths/entities/items/labels/singleLang.json [hunk 4]
```diff
@@ -11,5 +11,26 @@
...
+        "responses": {
+            "200": { 
+                "description": "Wikibase Entity ID",
+                "content": {
+                    "application/json": {
+                        "schema": { "type": "string" }
+                    }
+                }
+             },
```
Comment (itamargiv) [inline paths/entities/items/labels/singleLang.json:31]: The response should probably be the fingerprint itself, in lack of a proper ID: `EntityLabel`

File: paths/entities/properties/aliases/list.json [hunk 5]
```diff
@@ -12,5 +12,20 @@
...
             "default": { "$ref": "../../../../responses.json#/UnexpectedError" }
         }
+    },
+    "put": {
+        "tags": ["properties"],
+        "summary": "Set/Replace aliases in multiple languages at once",
+        "parameters": [
+            { "$ref": "../../../../parameters.json#/entityId" }
+        ],
+        "requestBody": { "$ref": "../../../../requests.json#/Property" },
```
Comment (itamargiv) [inline paths/entities/properties/aliases/list.json:22]: Same as aliases above

File: paths/entities/properties/aliases/singleLang.json [hunk 6]
```diff
@@ -13,5 +13,21 @@
...
         }
+    },
+    "put": {
+        "tags": ["properties"],
+        "summary": "Add / Replace a whole alias list by language",
+        "parameters": [
+            { "$ref": "../../../../parameters.json#/entityId" },
+            { "$ref": "../../../../parameters.json#/lang" }
+        ],
+        "requestBody": { "$ref": "../../../../requests.json#/Property" },
```
Comment (itamargiv) [inline paths/entities/properties/aliases/singleLang.json:24]: Same as above

File: paths/entities/properties/descriptions/singleLang.json [hunk 7]
```diff
@@ -11,5 +11,26 @@
...
         }
+    },
+    "put": {
+        "tags": ["properties"],
+        "summary": "Add / Replace a description to an entity by language",
+        "parameters": [
+            { "$ref": "../../../../parameters.json#/entityId" },
+            { "$ref": "../../../../parameters.json#/lang" }
+        ],
+        "requestBody": { "$ref": "../../../../requests.json#/Property" },
```
Comment (itamargiv) [inline paths/entities/properties/descriptions/singleLang.json:22]: Same as item descriptions

File: paths/entities/properties/descriptions/singleLang.json [hunk 8]
```diff
@@ -11,5 +11,26 @@
...
+        "responses": {
+            "200": { 
+                "description": "Wikibase Entity ID",
+                "content": {
+                    "application/json": {
+                        "schema": { "type": "string" }
+                    }
+                }
+             },
```
Comment (itamargiv) [inline paths/entities/properties/descriptions/singleLang.json:31]: Same as item descriptions

File: paths/entities/properties/labels/singleLang.json [hunk 9]
```diff
@@ -11,5 +11,26 @@
...
         }
+    },
+    "put": {
+        "tags": ["properties"],
+        "summary": "Add / Replace a label to an entity by language",
+        "parameters": [
+            { "$ref": "../../../../parameters.json#/entityId" },
+            { "$ref": "../../../../parameters.json#/lang" }
+        ],
+        "requestBody": { "$ref": "../../../../requests.json#/Property" },
```
Comment (itamargiv) [inline paths/entities/properties/labels/singleLang.json:22]: same as item labels

File: paths/entities/properties/labels/singleLang.json [hunk 10]
```diff
@@ -11,5 +11,26 @@
...
+        "responses": {
+            "200": { 
+                "description": "Wikibase Entity ID",
+                "content": {
+                    "application/json": {
+                        "schema": { "type": "string" }
+                    }
+                }
+             },
```
Comment (itamargiv) [inline paths/entities/properties/labels/singleLang.json:31]: Same as item labels

**Comment by guergana** [comment]
Closing, will start new PR after schemas update.

**Pull request closed without merging**